katello/katello-change-hostname

```diff
@@ -217,6 +217,10 @@ STDOUT.puts installer_output
 STDOUT.puts "Regenerating ueber certs"
 `foreman-rake katello:regenerate_ueber_certs`
 
+STDOUT.puts "Restarting puppet services"
+`service puppet restart`
+`service puppetserver restart`
```

Comment:
@sean797 That is true, but this script is given out to use on older versions of Katello, so I used `service` to still support them.

I'm unsure about that. I had to restart puppetserver on 3.3, maybe @ehelms knows more